src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicPlatformDetectionTests.cs

```diff
@@ -57,8 +57,6 @@ public async Task SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue()
             }
         }
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/82154", typeof(PlatformDetection), nameof(PlatformDetection.IsPpc64leProcess))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/82154", typeof(PlatformDetection), nameof(PlatformDetection.IsUbuntu2004), nameof(PlatformDetection.IsS390xProcess))]
```

Comment:
You can delete `IsUbuntu2004` property definition as well. This is its last use.